scripts/zones/West_Ronfaure/npcs/Laillera.lua

```diff
@@ -4,13 +4,18 @@
 -- !pos -127.297 -62.000 266.800 100
 -----------------------------------
 local ID = require("scripts/zones/West_Ronfaure/IDs")
+require("scripts/globals/quests")
 -----------------------------------
 
 function onTrade(player, npc, trade)
 end
 
 function onTrigger(player, npc)
-    player:showText(npc, ID.text.LAILLERA_DIALOG)
+    if player:getQuestStatus(SANDORIA, tpz.quest.id.sandoria.THE_PICKPOCKET) == QUEST_ACCEPTED then
+        player:showText(npc, ID.text.LAILLERA_DIALOG + 4)
```

Comment:
Anytime you have to use showtext instead of messageSpecial, and the npc is -not- a shop, it is all but guaranteed to be the incorrect way to display the dialog. This is likely to be an event that our extractor tool was unable to handle.

Reply:
Yeah, no extracts for this one :(